src/common/utils.js

```diff
@@ -55,7 +55,7 @@ function injectPageScript(url) {
     const s = document.createElement('script');
     s.src = url;
     s.charset = "UTF-8";
-    s.onload = () => s.remove();
+    // s.onload = () => s.remove();
```

Comment:
Any reason to keep the tag? Honestly, I just copied that from the stackoverflow, it never caused any issues but I don't see a reason for the tag to be removed either.

Reply:
Bot specifically, I think I just removed to check something and format to undo the change, will fix it tomorrow morning.